prow/cluster/monitoring/mixins/grafana_dashboards/boskos.jsonnet

```diff
@@ -0,0 +1,64 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+
+
+local dashboardConfig = {
+        uid: 'wSrfvNxWz',
+    };
+
+dashboard.new(
+        'Boskos Resource Usage',
+        time_from='now-1d',
+        schemaVersion=18,
+      )
+.addPanels([
+    graphPanel.new(
+        '%s usage status' % resource.friendly,
+        description='The number of %ss in each state.' % resource.friendly,
+        datasource='prometheus',
+        aliasColors={'busy': '#0000ff', 'cleaning': '#00eeff', 'dirty': '#ff8000', 'free': '#00ff00', 'leased': '#ee00ff', 'other': '#ff0000'}
+    )
+    .addTarget(prometheus.target(
+        '%s_busy' % resource.metric,
```

Comment:
These are the older metrics -- I would suggest

```
sum(boskos_resources{type="<my-named-thing>"}) by (state)
```

Reply:
We don't have that metric in our data... Sounds like our boskos instance isn't being updated properly? I'll open an issue.

Reply:
ref: https://github.com/kubernetes/test-infra/issues/11957

Reply:
Whoa. That was added ages ago. I thought `boskos` was automatically rolled out these days?